### ISSUE TITLE
a11y: Project management releases table

### ISSUE THREAD

**Issue by woodruffw** [pypa/warehouse#6079]
Accessibility improvements can be made to `/manage/project/{project_name}/releases/`:

* The release `<table>` itself should have a `<caption>`, as the first child node. This is partially mitigated by the `<h2>` above
* The `<table>` should include `scope=col` or another appropriate scope for each `<th>` node
* For dates in the "Release date" column, use Jinja2's `localize` instead of `humanize` to insert proper ARIA tags
* The option dropdown buttons should have `type=button` to indicate that they don't reset or resubmit any form data

**Comment by nlhkabu** [comment]
> The option dropdown buttons should have type=button to indicate that they don't reset or resubmit any form data

New ticket opened: https://github.com/pypa/warehouse/issues/6310

> For dates in the "Release date" column, use Jinja2's localize instead of humanize to insert proper ARIA tags

@woodruffw - What do you mean by this?  We currently use `format_date()`  from https://github.com/pypa/warehouse/blob/master/warehouse/i18n/filters.py
